### ISSUE TITLE
Follow player spectate option

### ISSUE THREAD

**Issue by Hornwitser** [explosivegaming/factorio-olympics#53]
**Feature overview**
Name: follow-player
TL;DR: Some way to follow a player when spectating them

**Details**
When clicking on a player the players GUI as spectator you're teleported to where the player is.  But in the race that player quickly goes off screen and is hard to follow.  It would be very useful for spectator and especially streamers to be able to have the game follow the player selected in the menu.

**Comment by Cooldude2606** [comment]
Pr has been merged, closing this issue.
